### ISSUE TITLE
Not Working on iOS devices

### ISSUE THREAD

**Issue by modegargar2-1** [dlutton/flutter_tts#372]
## 🐛 Bug Report

The package works perfectly fine in android devices, but when I start the speak method in an iOS device it doesn't start at all. 

### Expected behavior:
To start the speech when clicked on button that invokes speak() method. 

### Reproduction steps:
Use the example app provided by package, 
Build the application targeting latest iOS devices 
try the speak()  method

**Comment by dlutton** [comment]
@modegargar2-1 do you have your sound on your device on?  Are there any error messages that you receive when invoking speak()?

**Comment by modegargar2-1** [comment]
Yes, I have the sound on. There are no error message as this does not occur during debug mode.
The issue got self resolved after sometime. But this has occurred a few times. TTS does not provide sound on few devices of similar configuration for sometime.

For eg: for iOS 16.3 and 16.2 devices in India this issue started day before yesterday and got self resolved yesterday

**Comment by dlutton** [comment]
@modegargar2-1 this is common with devices depending on different situations.  You may need to set a category so that it works regardless of the user having their device audio on/off and other factors.  Below is just an example.

```
await flutterTts.setIosAudioCategory(IosTextToSpeechAudioCategory.ambient,
     [
          IosTextToSpeechAudioCategoryOptions.allowBluetooth,
          IosTextToSpeechAudioCategoryOptions.allowBluetoothA2DP,
          IosTextToSpeechAudioCategoryOptions.mixWithOthers
     ],
     IosTextToSpeechAudioMode.voicePrompt
);
```

**Comment by nakochan** [comment]
I also have this problem.
on iOS 16.2 - iphone 13 pro simulator

**Comment by dlutton** [comment]
@nakochan the newer emulators may not have TTS or languages come installed.  This is a known issue.  There shouldn't be an issues on physical devices.

**Comment by nakochan** [comment]
> 

Thank you. No problem on real device.
